inst/components/fill.scss

```diff
@@ -22,8 +22,18 @@
   width: 100%;
   height: 100%;
   margin: 0;
-  padding: 1rem;
-  gap: 1rem;
+  padding: $spacer;
+  gap: $spacer;
+}
+
+.bslib-page-title {
+  background-color: $bslib-page-title-bg;
+  color: $bslib-page-title-color;
+  font-size: $h4-font-size;
+  font-weight: 300;
+  padding: $spacer * 0.5;
+  padding-left: $bslib-sidebar-padding;
+  margin-bottom: 0;
```

Comment:
Overall this is great! Given that both Bootstrap provides [three navbar color schemes](https://getbootstrap.com/docs/5.3/components/navbar/#color-schemes) and that Bootswatch highlights these in their theme previews, I think we might want to think about leaving some room for building on those classes in the future.

Maybe we could [take inspiration from quarto](https://quarto.org/docs/websites/website-navigation.html#top-navigation), who IIUC add some [sass variables](https://github.com/quarto-dev/quarto-cli/blob/28457e739a6fd1b3b1ae1bebe736aa9a487e1e84/src/resources/formats/html/bootstrap/_bootstrap-variables.scss#L41-L56) around navbar color and hook off of that. Maybe we'd hook into a sass variable that could be set easily via `bs_theme()`?

I think we're on a reasonable path that would allow for this kind of expansion, but I also think we can reasonably expect users to ask for something like this soon.

Reply:
> Maybe we could [take inspiration from quarto](https://quarto.org/docs/websites/website-navigation.html#top-navigation), who IIUC add some [sass variables](https://github.com/quarto-dev/quarto-cli/blob/28457e739a6fd1b3b1ae1bebe736aa9a487e1e84/src/resources/formats/html/bootstrap/_bootstrap-variables.scss#L41-L56) around navbar color and hook off of that. Maybe we'd hook into a sass variable that could be set easily via bs_theme()?

I'm pretty sure Quarto took inspiration from bslib :) We also have a [`$navbar-bg`](https://rstudio.github.io/bslib/articles/bs5-variables.html#navbar-bg), but I'm not sure I want to tie that variable to `page_sidebar()`. At least, for right now, folks who want to theme it can do `bs_theme("bslib-page-title-bg" = "black")`

Reply:
Good point, Quarto actually got the `$navbar-bg` from bslib :)